spec/lib/binary-buffer-module.spec.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+describe("binaryBuffer", () => {
+    var binaryBuffer, copy;
+
+    beforeEach(() => {
+        binaryBuffer = require("../../lib/binary-buffer")();
+    });
+    [
+        "some string",
+        "",
+        new Buffer("some buffer"),
+        123
```

Comment:
Is there ever a case where you'd want an input that violates the jsdoc (besides if you have error handling/are testing error handling)?